ansible_commands/shc_ready.py

```diff
@@ -19,16 +19,16 @@ def __init__(self, module):
         self.user = module.params["spl_user"]
         self.password = module.params["spl_pass"]
         self.retry_times = module.params["retry_times"]
+        self.shc_bootstrap_delay = module.params["shc_bootstrap_delay"]
 
     def run(self):
         URL = "https://{0}:8089/services/shcluster/status?output_mode=json".format(self.captain_url)
         peers_online = False
         peers_ready = False 
-        time.sleep(60) # give the SHC some initial time to setup
         online_peers = None 
         for _ in range(0, self.retry_times):
-            time.sleep(30) # wait some time before executing next loop
             try:
+                time.sleep(self.shc_bootstrap_delay) # wait some time before executing next loop
```

Comment:
Note: This is still at the top of the loop, I'd put the sleep right after the exception handler (currently after line 45).